xshop/products/models.py

```diff
@@ -42,3 +43,9 @@ def save(self, *args, **kwargs):
         ean.write(buffer)
         self.barcode.save("barcode.png", File(buffer), save=False)
         return super().save(*args, **kwargs)
+
+
+class Rating(models.Model):
+    product = models.ForeignKey(Product, default=None, on_delete=models.PROTECT)
+    user = models.ForeignKey(User, default=None, on_delete=models.PROTECT)
+    rating = models.FloatField()
```

Comment:
Make it from 0 to 5 with validation
`MinValueValidator(0), MaxValueValidator(5)`

like this
![image](https://user-images.githubusercontent.com/6496157/124920782-fd07e000-dff7-11eb-9111-ff5be06814d6.png)